plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -53,13 +97,21 @@ public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 	 * @return bool Whether the tag should be tracked in URL Metrics.
 	 */
 	private function process_img( OD_HTML_Tag_Processor $processor, OD_Tag_Visitor_Context $context ): bool {
-		$src = $this->get_valid_src( $processor );
-		if ( null === $src ) {
+		if ( ! $this->is_img_with_valid_src_and_srcset( $processor ) ) {
```

Comment:
As noted below, this may be too conservative. We could still process this `IMG`. It's just we would skip any parts related to preloading the `src` or `srcset`.